{
  "uid" : "db0df7af6300c077",
  "name" : "NHS Number: Searching with invalid NHS number shows an error [9753108642-Enter a correct NHS number]",
  "fullName" : "/app/features/find_a_patient.feature:NHS Number: Searching with invalid NHS number shows an error",
  "historyId" : "4cb2e2253d26b616069dd8a9fc22175b",
  "time" : {
    "start" : 1728490546925,
    "stop" : 1728490564693,
    "duration" : 17768
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: assert False is True\n +  where False = check_nhs_number_error_message_text_exists('Enter a correct NHS number')",
  "statusTrace" : "fixturefunc = <function step_error_message_appears_for_nhs_number at 0x7f5a1652fe20>\nrequest = <FixtureRequest for <Function test_nhs_number_searching_with_invalid_nhs_number_shows_an_error[9753108642-Enter a correct NHS number]>>\nkwargs = {'errorMessage': 'Enter a correct NHS number'}\n\n    def call_fixture_func(\n        fixturefunc: _FixtureFunc[FixtureValue], request: FixtureRequest, kwargs\n    ) -> FixtureValue:\n        if is_generator(fixturefunc):\n            fixturefunc = cast(\n                Callable[..., Generator[FixtureValue, None, None]], fixturefunc\n            )\n            generator = fixturefunc(**kwargs)\n            try:\n                fixture_result = next(generator)\n            except StopIteration:\n                raise ValueError(f\"{request.fixturename} did not yield a value\") from None\n            finalizer = functools.partial(_teardown_yield_fixture, fixturefunc, generator)\n            request.addfinalizer(finalizer)\n        else:\n            fixturefunc = cast(Callable[..., FixtureValue], fixturefunc)\n>           fixture_result = fixturefunc(**kwargs)\n\n.tox/py311/lib/python3.11/site-packages/_pytest/fixtures.py:898: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nerrorMessage = 'Enter a correct NHS number'\n\n    @then(parse('I can see an nhs number error message {errorMessage}'))\n    def step_error_message_appears_for_nhs_number(errorMessage):\n        attach_screenshot(\"error_message_appears_for_nhs_number\")\n>       assert check_nhs_number_error_message_text_exists(errorMessage) is True\nE       AssertionError: assert False is True\nE        +  where False = check_nhs_number_error_message_text_exists('Enter a correct NHS number')\n\ntests/test_find_a_patient_steps.py:126: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : false,
  "newPassed" : false,
  "retriesCount" : 0,
  "retriesStatusChange" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: assert False is True\n +  where False = check_nhs_number_error_message_text_exists('Enter a correct NHS number')",
    "statusTrace" : "fixturefunc = <function step_error_message_appears_for_nhs_number at 0x7f5a1652fe20>\nrequest = <FixtureRequest for <Function test_nhs_number_searching_with_invalid_nhs_number_shows_an_error[9753108642-Enter a correct NHS number]>>\nkwargs = {'errorMessage': 'Enter a correct NHS number'}\n\n    def call_fixture_func(\n        fixturefunc: _FixtureFunc[FixtureValue], request: FixtureRequest, kwargs\n    ) -> FixtureValue:\n        if is_generator(fixturefunc):\n            fixturefunc = cast(\n                Callable[..., Generator[FixtureValue, None, None]], fixturefunc\n            )\n            generator = fixturefunc(**kwargs)\n            try:\n                fixture_result = next(generator)\n            except StopIteration:\n                raise ValueError(f\"{request.fixturename} did not yield a value\") from None\n            finalizer = functools.partial(_teardown_yield_fixture, fixturefunc, generator)\n            request.addfinalizer(finalizer)\n        else:\n            fixturefunc = cast(Callable[..., FixtureValue], fixturefunc)\n>           fixture_result = fixturefunc(**kwargs)\n\n.tox/py311/lib/python3.11/site-packages/_pytest/fixtures.py:898: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nerrorMessage = 'Enter a correct NHS number'\n\n    @then(parse('I can see an nhs number error message {errorMessage}'))\n    def step_error_message_appears_for_nhs_number(errorMessage):\n        attach_screenshot(\"error_message_appears_for_nhs_number\")\n>       assert check_nhs_number_error_message_text_exists(errorMessage) is True\nE       AssertionError: assert False is True\nE        +  where False = check_nhs_number_error_message_text_exists('Enter a correct NHS number')\n\ntests/test_find_a_patient_steps.py:126: AssertionError",
    "steps" : [ {
      "name" : "Given I am on the find a patient by nhs number page",
      "time" : {
        "start" : 1728490546925,
        "stop" : 1728490554619,
        "duration" : 7694
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And I enter 9753108642 as the nhs number",
      "time" : {
        "start" : 1728490554619,
        "stop" : 1728490563179,
        "duration" : 8560
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When I click the search button",
      "time" : {
        "start" : 1728490563179,
        "stop" : 1728490564552,
        "duration" : 1373
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Then I can see an nhs number error message Enter a correct NHS number",
      "time" : {
        "start" : 1728490564553,
        "stop" : 1728490564693,
        "duration" : 140
      },
      "status" : "failed",
      "statusMessage" : "assert False is True\n +  where False = check_nhs_number_error_message_text_exists('Enter a correct NHS number')",
      "statusTrace" : "AssertionError: assert False is True\n +  where False = check_nhs_number_error_message_text_exists('Enter a correct NHS number')\n",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "a6b04db35a627348",
        "name" : "data/attachments/dev_chrome_129.0.6668.100_error_message_appears_for_nhs_number.png",
        "source" : "a6b04db35a627348.png",
        "type" : "image/png",
        "size" : 66892
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 4,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "host",
    "value" : "40090a47a729"
  }, {
    "name" : "thread",
    "value" : "1043-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest-bdd"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "feature",
    "value" : "Find a patient"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "errorMessage",
    "value" : "Enter a correct NHS number"
  }, {
    "name" : "nhsNumber",
    "value" : "9753108642"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "db0df7af6300c077.json",
  "parameterValues" : [ "Enter a correct NHS number", "9753108642" ]
}